src/bootstrap.js

```diff
@@ -14,10 +18,25 @@ function install() {} // eslint-disable-line no-unused-vars
 function uninstall() {} // eslint-disable-line no-unused-vars
 
 function startup() { // eslint-disable-line no-unused-vars
+  Services.prefs.setBoolPref(UA_ENABLE_PREF_NAME, true);
```

Comment:
Experience from other features, when in developing stage, it's better not to directly turned on. Developers can easily turned on by 1) setup about:config or 2) patch preference file.
When code is checked-in to m-c, and it's not completed or verified by QA, at least we can promise it will not affect other features.
After implementation completed and ready for QA, we can submit a patch to user.js to turn on.
If we still prefer directly turn on in code, please add a nightly or developer edition detection check to prevent it will not affect beta and official release.
@denschub It will prevent in-complete implementation checked-in to m-c and affect something we don't want. I agree to turn on this feature in Dev Edition and Nightly, but please do this after implementation complete and approved by release manager. Does this make sense?